wicketd/src/http_entrypoints.rs

```diff
@@ -48,6 +53,12 @@ pub fn api() -> WicketdApiDescription {
     api
 }
 
+#[derive(Clone, Copy, Debug, Serialize, Deserialize, JsonSchema, PartialEq)]
+pub struct ForceRefresh {
+    /// If true, refresh inventory from MGS instead of returning cached data.
+    pub force_refresh: bool,
```

Comment:
Realized that this should probably be a list of SPs that we want to force refresh.

Reply:
Good idea; changed in b3976eac. Since `wicketd` is currently using the bulk "get all SPs" endpoint it doesn't actually use the list (other than "empty or not"), but we can start using it in a future PR.